core/domain/state_domain.py

```diff
@@ -961,7 +965,11 @@ class RecordedVoiceovers(python_utils.OBJECT):
     """
 
     def __init__(self, voiceovers_mapping):
-        """Initializes a RecordedVoiceovers domain object."""
+        """Initializes a RecordedVoiceovers domain object.
+
+        Args:
+            voiceovers_mapping: Voiceovers. Voiceovers domain object.
```

Comment:
@DubeySandeep what do you think is the type of `voiceovers_mapping`?

Reply:
@edeediong You will need to investigate this, as part of writing this PR. (See a previous comment i wrote above.)

I agree with @DubeySandeep that it cannot be `Voiceovers` because that is neither a standard Python type, nor do we have a Voiceovers class in our codebase.

Reply:
Still working on this though